### ISSUE TITLE
Bug with xcodeindentation for chained operators

### ISSUE THREAD

**Issue by malonehedges** [nicklockwood/SwiftFormat#858]
When performing many operations on some value, Xcode will keep them all at the same indentation level. The new update including the fix for #848 causes each line to be further indented. The behavior in 848 is specific to guard (and maybe if?) statements, but when calling many functions/properties in an assignment they shouldn't be indented.

Expected indentation:
```swift
let indexPath = self.collectionView.indexPathsForVisibleItems
  .filter { _ in
    true
  }
  .compactMap { _ -> String? in
    "hello"
  }
  .filter { _ in
    false
  }
  .sorted { _, _ in
    false
  }
  .first
```

With `xcodeindentation` enabled:
```swift
let indexPath = self.collectionView.indexPathsForVisibleItems
  .filter { _ in
    true
  }
    .compactMap { _ -> String? in
      "hello"
    }
      .filter { _ in
        false
      }
        .sorted { _, _ in
          false
        }
          .first
```

**Comment by mthole** [comment]
I see something similar in my codebase. It looks like this regressed in 0.47.11-- likely related to the "bugfix" mentioned for `--xcodeindention` in the release notes. Downgrading to 0.47.10 fixes my particular use case.

**Comment by nicklockwood** [comment]
@malonehedges @mthole fixed in 0.47.12
